tests/functional/rhcs_setup.yml

```diff
@@ -27,22 +27,10 @@
         dest: "{{ change_dir }}/vagrant_variables.yml"
       when: change_dir is defined
 
-    - name: change ceph_stable to False
+    - name: change ceph_origin to distro
       replace:
-        regexp: "ceph_stable:.*"
-        replace: "ceph_stable: False"
-        dest: "{{ change_dir }}/group_vars/all"
-      when: change_dir is defined
-
-    - name: set ceph_rhcs to True
-      lineinfile:
-        line: "ceph_rhcs: True"
-        dest: "{{ change_dir }}/group_vars/all"
-      when: change_dir is defined
-
-    - name: set ceph_origin to distro
-      lineinfile:
-        line: "ceph_origin: distro"
+        regexp: "ceph_origin:.*"
+        replace: "ceph_origin: distro"
```

Comment:
We need to support the scenario where a ``rhcs`` user wants to use Satellite or some other means of hosting the ``rhcs`` packages themselves. In this case they've prepared all their nodes with a custom repo file to consume those self-hosted packages. 

This is also a very common install method for downstream QE when testing unreleased packages.